html/login.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>로그인</title>
+    <link rel="icon" type="image/svg" href="/image/logo.svg" />
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
+    />
+    <link rel="stylesheet" href="/css/auth.css" />
+  </head>
+
+  <body class="body_login">
+    <header>
+      <a href="/">
+        <img class="logo" src="/image/logo.svg" alt="판다마켓 로고" />
+      </a>
+      <a class="logo-title" href="/">
+        <img src="/image/logo_title.svg" alt="판다마켓 로고 글씨" />
+      </a>
+    </header>
+
+    <main>
+      <form>
+        <div class="input-box">
+          <label for="email">이메일</label>
+          <input name="email" id="email" type="email" />
+        </div>
+
+        <div class="input-box">
+          <label for="password">비밀번호</label>
+          <div class="pw-content">
+            <input name="password" id="password" type="password" />
+            <img
+              class="eye_icon"
+              src="/image/password_toggle_icon.svg"
+              alt="비밀번호를 보이지 않게 하는 눈 아이콘"
+            />
+          </div>
+        </div>
+
+        <button>로그인</button>
```

Comment:
💬 여담
button 태그의 type 속성을 명시하지 않으면 submit을 기본값으로 가지게 됩니다~
이러한 type 속성은 button 태그에 중요한 요소이므로 명시적으로 작성해주는 것을 추천드려요.

```suggestion
        <button type="submit">로그인</button>
```

+type이 submit인 button의 경우 form에서 제출 이벤트가 발생하면, 해당 요소의 onclick 이벤트가 실행되게 됩니다.
이를 더 풀어서 말하자면 form 태그안의 input, button 과 같은 상호작용 요소에서 submit 이벤트, enter key 입력이 발생하면 해당 form 에서의 submit 이벤트가 실행된다는 것입니다.
자세한 것은 추후 학습 하시면서 배우실 것이니 우선은 button 태그의 type 속성을 명시적으로 작성하는 것이 좋다는 것만 기억해주시면 좋겠습니다~

[mdn button](https://developer.mozilla.org/ko/docs/Web/HTML/Element/button#type)